### ISSUE TITLE
Group Block: Add block justification setting to the layouts tab

### ISSUE THREAD

**Issue by andersnoren** [WordPress/gutenberg#30559]
In Full Site Editing themes, the group block allows you to modify the width of blocks contained in group blocks, either by specifying a width manually or by checking "Inherit default layout", which sets the width of the child blocks to the default content widths in the theme.

<img width="1458" alt="image" src="https://user-images.githubusercontent.com/8181091/113849492-d9838e00-9799-11eb-80af-fc73ea1cd2b1.png">

Currently, the child blocks are always given a horizontal margin of auto, centering them in the group block they're in. I think it would be useful to add the option to choose justification of the child blocks in the group, so you can choose whether to set them to the left, center or right of the group, with center being the default.

For example, if you have a columns block with each column containing an image, a heading and text, you might want to limit the width of the heading and paragraph to preserve readability. But if you wrap the heading and the text in a group block and set a width limit, the heading and text become centered in the column.

<img width="1950" alt="image" src="https://user-images.githubusercontent.com/8181091/113850384-b1485f00-979a-11eb-887e-7440a9c81bcd.png">

Some might prefer this, of course, but those who would rather have the heading and text align with the left or right side of the column currently have no way of doing that. Another common example is headings justified to the left of the outer edge of the content, like this:

<img width="1014" alt="image" src="https://user-images.githubusercontent.com/8181091/113852971-6c71f780-979d-11eb-9531-bbcf734fa2c8.png">

Which can't be replicated out of the box currently, since the child blocks of the group are always centered:

<img width="1410" alt="image" src="https://user-images.githubusercontent.com/8181091/113853181-a8a55800-979d-11eb-9499-dde342a763f3.png">

TLDR; adding a child block justification setting for groups would add a lot more flexibility when creating layouts.

**Comment by jlavoie13** [comment]
While I like the addition of this new feature I've had to abandon the use of it for this reason. Because of the inline styles there's no easy way to override it.

**Comment by mtias** [comment]
This should be covered now:

<img width="350" height="123" alt="Image" src="https://github.com/user-attachments/assets/da806575-f64c-489d-a376-100fdd35f7cd" />
